### PR TITLE
custom audio driver

### PR DESCRIPTION
**Pull request by abdulajet**
I moved the min ios version to ios 12, cleaned up all the project warnings and deprecations that arose too.

PR relies on #220, review the latest commit

### CODE REVIEW

File: Custom-Audio-Driver/External-Audio-Device/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -2,68 +2,93 @@
   "images" : [
     {
       "idiom" : "iphone",
-      "size" : "29x29",
```
Comment (jvalli) [inline Custom-Audio-Driver/External-Audio-Device/Images.xcassets/AppIcon.appiconset/Contents.json:5]: Here seems there are no changes, just the order of the json content. If that is the case, please remove it from the PR.

Comment (abdulajet) [reply Custom-Audio-Driver/External-Audio-Device/Images.xcassets/AppIcon.appiconset/Contents.json:5]: the resolutions are different

Comment (jvalli) [reply Custom-Audio-Driver/External-Audio-Device/Images.xcassets/AppIcon.appiconset/Contents.json:5]: Which ones? I see a few new ones, but nothing is changed or removed. What Xcode version did you use?

Comment (abdulajet) [reply Custom-Audio-Driver/External-Audio-Device/Images.xcassets/AppIcon.appiconset/Contents.json:5]: oh wait yes I see what you mean now. Will remove 👍

**Review by jvalli — APPROVED** [review]

**Pull request merged**